src/crypto/TS.h

```diff
@@ -23,6 +23,8 @@
 #include <memory>
 
 typedef struct pkcs7_st PKCS7;
+typedef struct CMS_ContentInfo_st CMS_ContentInfo;
+typedef struct TS_tst_info_st TS_TST_INFO;
```

Comment:
Why the typedefs instead of direct use? It is generally considered a mediocre practice.